journal/modernized/18-chapter-15.adoc

```diff
@@ -134,14 +137,16 @@ Thus have loving-kindness and tender mercy still followed me;
 and I actually feel better in health than when I left London,
 though my legs and hands are somewhat swelled, probably with the cold.
 
+--
+
 [.offset]
 +++[+++He returned to this country in the Sixth Month following,
 where he was shortly afterwards joined by our brother Charles,
 who had believed it his duty to offer himself as a companion to his father,
 and was finally accepted in that capacity by the Meeting for Sufferings.
 In a letter written on this subject, dated the 2nd of Sixth Month, 1833,
 Charles Wheeler says,
-
+//does this need to be signaled as a letter? These offsets are tricky.
```

Comment:
This is a particularly weird one, but no, i think, let's leave it as is. so you can remove this comment and restore the blank line here.